assets/stories/skate-design/Typography.mdx

```diff
@@ -0,0 +1,126 @@
+import { Meta, Source, Unstyled } from "@storybook/blocks"
+import "../../css/storybook_reboot.scss"
+
+<Meta title="Typography" />
+
+# Global Settings / Style
+
+Examples of typography styles found in Skate, based on defaults from
+Reboot with some modifications. This roughly follows the outline of
+the <a href="https://getbootstrap.com/docs/5.3/content/typography/">
+Bootstrap typography documentation</a> adapted specifically for Skate.
+The sections that follow below are all rendered with Skate, as opposed
+to Storybook, styling.
+
+<Unstyled>
+  <p>
+    {
+      "The default font is Inter. Body text is 1rem, 400 weight, $gray-900(#141524)."
+    }
+    <br />
+    {"In paragraphs, we use 150% line height."}
```

Comment:
suggestion: It's a little confusing to go from reading about the font to talking about the line height. Could we move the font text into it's own paragraph and copy the text from figma?
```suggestion
  The default font is Inter. Body text is 1rem, 400 weight, $gray-900(#141524).

  <p>
    {"Paragraph text. When body text is"}
    <br />
    {"in paragraphs, we use 150% line height."}
```

I'd also be happy with using the text from [bootstrap's line-height docs](https://getbootstrap.com/docs/5.3/utilities/text/#line-height), instead of trying for force a break within text. It might be good to have the different line-height utilities shown anyway?